src/components/sidebarcom.jsx

```diff
@@ -1,133 +1,185 @@
+      );
+    } else {
+      seticonstate(
+        <AiFillHeart className="w-5 h-5 absolute ml-3 pointer-events-none" />
+      );
     }
-    else if(array[1] === "js" || array[1] === "javascript"){
-        seticonstate(<DiJavascript1 className="w-5 h-5 absolute ml-3 pointer-events-none" />)
-    }
-    else if (array[1] === undefined ){
-        seticonstate(<GiBrokenHeart className="w-5 h-5 absolute ml-3 pointer-events-none" />)
-    }
-    else if (array[1] === "rs" || array[1] === "rust"){
-        seticonstate(<DiRust className="w-5 h-5 absolute ml-3 pointer-events-none" />)
-    }
-    else if ( array[1] === "c#"){
-        seticonstate(<SiCsharp className="w-5 h-5 absolute ml-3 pointer-events-none" />)
-    }
-    else{
-        seticonstate(<AiFillHeart className="w-5 h-5 absolute ml-3 pointer-events-none" />)
-    }
-    
-}
-
+  };
 
-    
-    return (
-        <aside className="w-64" aria-label="Sidebar">
-   <div className="overflow-y-auto  py-5 px-3 bg-[#2A4867] rounded min-h-[80vh] min-w-[190vh] mr-2 ml-1">
-    <ul id="unorder" >
-    {[...Array(tabCount)].map((i) => {
-        
-        
-        return(
-            
-            <li className=" flex p-2" key={i}>
+  return (
+    <aside className="w-64" aria-label="Sidebar">
+      <div className="overflow-y-auto  py-5 px-3 bg-[#2A4867] rounded min-h-[80vh] min-w-[190vh] mr-2 ml-1">
+        <ul id="unorder">
+          {[...Array(tabCount)].map((i) => {
+            return (
+              <li className=" flex p-2" key={i}>
                 <div className="relative flex items-center text-gray-400 focus-within:text-gray-600">
-                <input type="text" id="textinput" onChange={changingfunc}  className="pr-3 pl-10 bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500" placeholder="untitled" />
-                {/* {arrary.map((value) => 
+                  <input
+                    type="text"
+                    id="textinput"
+                    onChange={changingfunc}
+                    className="pr-3 pl-10 bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"
+                    placeholder="untitled"
+                  />
+                  {/* {arrary.map((value) => 
                 {
                     
                     let values = Object.values(value)
                     // console.log(values);
                     return values.map((value) => {return value});
                     })}; */}
-
-                {/* TO ITER THROUGH THE ARRAY AND BE ABLE TO FETCH TEH OBJECT */}
-                {arrary.map((value) => {
```

Comment:
how will u loop over the array of instance , without this ?

Reply:
You are not managing it as array properly, That's why i commented that to work theme part

Reply:
Please connect to me via discord killerthief#4626